src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1592,8 +1592,10 @@ def load_case_output_description(
 ) -> case_description.CaseOutputDescription:
```

Comment:
Should I add a unit test for this function somewhere?

Reply:
I believe it is already tested since you just added those new parameters to the `filled_case_description`.

Reply:
You'd think that, but the tests pass regardless of the two fields added to this dict.